Makefile

```diff
@@ -0,0 +1,12 @@
+NAME = skilldlabs/frontend
+TAGS = d7 latest omega_4_3 omega_4_4 zen
+
+.PHONY: all build push
+
+all: build push
```

Comment:
Any idea for testing? I think at least 2 goals in make file is enough so let's file separate PR for #16